ee/clickhouse/queries/funnels/funnel_trends.py

```diff
@@ -15,7 +15,7 @@
 HUMAN_READABLE_TIMESTAMP_FORMAT = "%a. %-d %b"
 
 
-class ClickhouseFunnelTrends(ClickhouseFunnelNew):
+class ClickhouseFunnelTrends(ClickhouseFunnelBase):
```

Comment:
Eventually, I think visualisation like Trends *should* inherit from the different types of ordered funnels, and use their now-new `get_step_counts_query()` as the seed query, but now's not the time for it. ( Want atleast a pattern of 2 before we generalise, so we don't create the wrong abstractions)

Reply:
yup, agreed